### PR TITLE
flamerobin: init at 0.9.3.1

### PR DESCRIPTION
**Pull request by uralbash**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [x] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/flamerobin/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, wxGTK30, boost, firebird }:
+
+stdenv.mkDerivation rec {
+  version = "0.9.3.1";
+  name = "flamerobin-${version}";
+
+  src = fetchurl {
```
Comment (zimbatm) [inline pkgs/applications/misc/flamerobin/default.nix]: for github urls it's better to use the following:
```
fetchFromGitHub {
  owner = "mariuz";
  repo = "flamerobin";
  rev = version;
  sha256 = "???";
};
```

File: pkgs/applications/misc/flamerobin/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  preBuild = ''
+    sed -i 's/CXXFLAGS = -g -O2/CXXFLAGS = -g -O2 -nostartfiles/' Makefile
+  '';
+
+  configureFlags = [
+    "--disable-debug"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "FlameRobin is a database administration tool for Firebird RDBMS";
```
Comment (zimbatm) [inline pkgs/applications/misc/flamerobin/default.nix]: It's better not to repeat the package name in the description to keep it short. And non-repetitive.

**Comment by zimbatm** [comment]
Thanks for your contribution. I just left to tiny fixes, just ping me when they are done.

**Comment by uralbash** [comment]
@zimbatm all fixed

**Pull request merged**